### DISCUSSION TITLE
工具箱>快速设置>Swap设置了一下交换分区大小导致无法开机

### DISCUSSION THREAD

**Discussion by tomseanmy** [1Panel-dev/1Panel#5720] [Q&A]
### 联系方式

tom_sean@outlook.com

### 1Panel 版本

v1.10.11-lts

### 问题描述

系统环境：debian12
在工具箱>快速设置>Swap设置了一下交换分区大小（值大于原始Swap分区大小）导致无法开机

我有两块盘/sda和/sdb组了一个lv_data逻辑卷，通过以上设置后导致原有得/sda变成了/sdc并且uuid也发生了变化导致lv_data启动无法挂载。从恢复模式进去系统后vgs看到原有的/sda的位置变成了unknow device 提示找不到对应uuid

### 重现步骤

工具箱>快速设置>Swap设置大小

### 期待的正确结果

正常变更Swap分区或者变更无效，不会导致无法开启或者改变其他物理卷盘符和uuid

**Comment by wanghe-fit2cloud** [comment]
在测试环境中未能复现上述问题，操作系统为 CentOS 7.9 和 Ubuntu 22.04。
我们后续也会在 Debian 12 操作系统上进行测试。
